src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c

```diff
@@ -3399,7 +3424,7 @@ void KaleidoScope_Update(PlayState* play) {
                             gSaveContext.buttonStatus[3] = BTN_ENABLED;
                         gSaveContext.hudVisibilityMode = HUD_VISIBILITY_NO_CHANGE;
                         Interface_ChangeHudVisibilityMode(HUD_VISIBILITY_ALL);
-#if OOT_NTSC
+#if OOT_NTSC && PLATFORM_GC
```

Comment:
nit: I'd write `#if PLATFORM_GC && OOT_NTSC` since the GC/N64 differences is more important IMO